test/result-list-spec.js

```diff
@@ -144,7 +146,7 @@ describe('ResultList', function() {
     });
 
     it('throws an error when there is no previous page', function() {
-      var test = function() {
+      let test = function() {
```

Comment:
`expect(() => resultList.previous()).to.throw('There is no previous page');` ?

I don't think it looks that good.